src/control-plane/alb-lambda-portal.ts

```diff
@@ -376,21 +376,23 @@ export class ApplicationLoadBalancerLambdaPortal extends Construct {
       ],
     );
 
-    const lambdaFn = new DockerImageFunction(this, 'portal_fn', {
+    const lambdaFn = new LambdaFunction(this, 'portal_fn', {
```

Comment:
Could we use `SolutionNodejsFunction` to share the configuration of Lambda?

Reply:
SolutionNodejsFunction not support `code: this.getLambdaCode()`